category.html

```diff
@@ -51,6 +51,26 @@
     <title>Download</title>
```

Comment:
Wrong stylesheet link `<link rel="stylesheet" href="css/download.css" />
Change it to href="css/category.css" 
`